packages/jsii-pacmak/test/generated-code/harness.ts

```diff
@@ -67,7 +67,7 @@ export function verifyGeneratedCodeFor(
 
       expect({ [TREE]: checkTree(outDir) }).toMatchSnapshot('<outDir>/');
 
-      if (targetName !== TargetName.PYTHON) {
+      if (targetName !== TargetName.PYTHON || process.env.SKIP_MYPY_CHECK) {
```

Comment:
The Mypy check adds between 40 and 100 seconds to the execution... Disabling it temporarily can help iterating much faster...